learn/getting-started/writing-a-grpc-service-with-ballerina.md

```diff
@@ -87,55 +108,66 @@ $ bal grpc --mode service --input helloworld.proto --output greeter_service/
 Once successfully executed, you will see the output below.
 
 ```bash
+Continuing with the existing protoc executor file at /var/folders/6p/8h4k83hj0r98n1s6mc3jy0480000gn/T/protoc-3.9.1-osx-x86_64.exe
```

Comment:
This output line varies depending on whether the executor is already downloaded or not. Here it is already downloaded. Shall we remove this line?

Reply:
Removed in https://github.com/ballerina-platform/ballerina-dev-website/pull/4069/commits/ebff86fbdafea4347d3b8a88f47610696e5787d9.